libs/payments/eligibility/src/lib/eligibility.manager.spec.ts

```diff
@@ -262,4 +287,431 @@ describe('EligibilityManager', () => {
       });
     });
   });
+
+  describe('getProductIdOverlap', () => {
+    it('should return empty result', async () => {
+      const mockTargetOffering = EligibilityContentOfferingResultFactory();
+      mockOfferingResult.getOffering = jest.fn().mockReturnValueOnce(undefined);
```

Comment:
nit. Just some cleanup, doesn't look like `mockOfferingResult.getOffering` needs to be mocked and the mocks can be removed in `describe('getProductIdOverlap')...`.

Reply:
Lets clean this up in a polish PR.